### DISCUSSION TITLE
Can't run Windows 11 ARM (Build 22533)

### DISCUSSION THREAD

**Discussion by PolarHacker** [utmapp/UTM#3727] [UTM Help (macOS)]
Hello! I am trying to run the ARM version of Windows 11. I joined the insider program and downloaded the .vhdx, but when I try to run it in UTM I am presented with the error attached.
![UTM Win 11 ARM Error](https://user-images.githubusercontent.com/10091313/156924561-af2e8e40-8fa8-438b-b721-0a155640bda7.png)

I also tried following the guide found [here](https://medium.com/macoclock/how-to-install-windows-11-on-an-m1-mac-for-free-94fb7a43eb47) after getting that error. It seems like it was designed for a slightly older version of UTM, but one of the steps is to convert the .vhdx file into .qcow2 using qemu in Terminal. I get this error after trying to run the command

**qemu-img convert -p -O qcow2 /Users/name/UTM/VMs/Windows11_InsiderPreview_Client_ARM64_en-us_22533.vhdx /Users/name/UTM/VMs/Windows11_InsiderPreview_Client_ARM64_en-us_22533.qcow2**

`qemu-img: VHDX BAT entry 1 start offset 9975103488 points after end of file (8385921024). Image has probably been truncated.
qemu-img: Could not open '/Users/name/UTM/VMs/Windows11_InsiderPreview_Client_ARM64_en-us_22533.vhdx': Could not open '/Users/name/UTM/VMs/Windows11_InsiderPreview_Client_ARM64_en-us_22533.vhdx': Invalid argument
`

I am using the Windows 11 Build 22533. I am on a 2020 M1 MacBook Air, 8GB RAM. Any help would be much appreciated!

**Comment by osy** [comment]
As the error indicated, your VHDX was truncated probably because your download was incomplete. Verify that you have downloaded the image correctly and try again.

On v3.x of UTM you no longer need to manually convert the VHDX, it will be done automatically.
